### DISCUSSION TITLE
How can I wrap the default blocks in my own HTML?

### DISCUSSION THREAD

**Discussion by dkrusenstrahle** [ueberdosis/tiptap#1308] [General Questions]
Is there a way to wrap all blocks (even the default ones like <p>) in custom code?
I need to let my users be able to set background color and so on, even for default blocks (I can do it for my own custom blocks).

For example the table block, if I could wrap in i could add color picker and table specific commands buttons for it.
What is the recommended way of wrapping for example the table block?

Thanks!

**Comment by philippkuehn** [comment]
you can try this: `commands.wrapIn('yourCustomNode')`
